### ISSUE TITLE
Can't import the css

### ISSUE THREAD

**Issue by alexanderhorner** [oleksiikhr/vue-stripe-menu#250]
**Describe the bug**
When trying to import the css I get an error.

```console
[vite] Internal server error: Missing "./dist/vue-stripe-menu.css" export in "vue-stripe-menu" package
```

```javascript
import 'vue-stripe-menu/dist/vue-stripe-menu.css'
```

I think you have to define it as an export or something like that (not an npm expert)

**Comment by alexanderhorner** [comment]
This is using the 3.0.0-beta.1 btw

**Comment by oleksiikhr** [comment]
Oh, you're too fast :D

I uploaded the beta, and I'll try after work to check it on Vue 3 and Nuxt. Bo I've been keeping this PR open for too long, and I feel like I've broken everything I can break :)

**Comment by alexanderhorner** [comment]
Will be fixed by #251
